src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -3297,8 +3303,9 @@ protected override bool HasChildItems(string path)
                 // If the above didn't throw an exception, check to
                 // see if we should proceed and if it contains any children
                 if ((directory.Attributes & FileAttributes.Directory) != FileAttributes.Directory)
+                {
                     return false;
-
+                }
```

Comment:
You should add another blank line after this, too, or CodeFactor will complain about that as well. 😄

Reply:
Will do.